wolfcrypt/src/asn.c

```diff
@@ -18742,6 +18756,41 @@ MimeParam* wc_MIME_find_param_attr(const char* attribute,
     return param;
 }
 
+/*****************************************************************************
+* wc_MIME_canonicalize - Canonicalize a line by converting all line endings
```

Comment:
Is the reason for converting to CRLF is so the hash will always compute the same?

Reply:
In a multipart message, the first part is the content that is hashed (which we are converting here) and the second part is a PKCS7 object which includes a hash of the content.  The hash is calculated using the canonicalized (CRLF converted) content, not the exact content included.  So we need to convert it to calculate the hash correctly.